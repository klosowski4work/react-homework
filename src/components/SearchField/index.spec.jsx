import React from 'react';
import { render } from 'enzyme';
import { SearchField } from './';

describe('SearchField component', () => {
    it('should render component', () => {
        const wrapper = render(<SearchField />);
        expect(wrapper).toMatchSnapshot();
    });
});