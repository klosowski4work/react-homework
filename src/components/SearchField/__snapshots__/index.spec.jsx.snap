// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField component should render component 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "search-field__input",
      "type": "text",
    },
    "children": Array [],
    "name": "input",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "type": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "type": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
